packages/golden_toolkit/pubspec.yaml

```diff
@@ -6,7 +6,7 @@ repository: https://github.com/eBay/flutter_glove_box/tree/master/packages/golde
 issue_tracker: https://github.com/eBay/flutter_glove_box/issues
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">=2.15.0 <3.0.0"
```

Comment:
@coreysprague - was this change required? I now can't use the latest golden toolkit because I haven't moved my large work project to 2.15.

I only wanted the `isTest` fix from the last pull request.

Guess I'll have to fork for time being and add that commit